molstar-extension/src/volumes-and-segmentations/entry-geometric-segmentation.ts

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
Move `csv-extension`, `meshes`, `volumes-and-segmentations` to `extension/*` folder.

Reply:
@dsehnal, you mean directly in `molstar-extension`?

Reply:
- `molstar-extension/src/extension/*` will have all the extension-related code
- `molstar-extension/src/viewer/*` will have the viewer

Reply:
Ok, thanks